docs/coverage.md

```diff
@@ -1,12 +1,12 @@
-#### [Safe Apps liveness](./../src/safe-apps/working_apps.test.js)
+#### [Safe Apps liveness](./../src\safe-apps/working_apps.test.js)
```

Comment:
Please don't merge it like this. With is it just one slash that got windows'ified and not others?

Reply:
Sorry I overlooked this. Thanks for the catch